packages/taquito-local-forging/src/constants.ts

```diff
@@ -25,8 +25,12 @@ export enum CODEC {
   INT16 = 'int16',
   PARAMETERS = 'parameters',
   ADDRESS = 'address',
+  SMART_ROLLUP_ADDRESS = 'smart_rollup_address',
   SMART_CONTRACT_ADDRESS = 'smart_contract_address',
+  SMART_ROLLUP_CONTRACT_ADDRESS = 'smart_rollup_contract_address',
```

Comment:
probably better to group smart_rollup_address and smart_rollup_contract_address to be adjacent to each other. A nitpick but it's more logical to do so.